.teamcity/_self/projects/DesktopApp.kt

```diff
@@ -0,0 +1,166 @@
+package _self.projects
+
+import _self.bashNodeScript
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.ParameterDisplay
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+
+object DesktopApp : Project({
```

Comment:
This is a copy/paste from the same project in `settings.kts`, plus the change of `id("WpDesktop")`